app/recurring_job.go

```diff
@@ -376,6 +410,13 @@ func (job *Job) doSnapshot() (err error) {
 }
 
 func (job *Job) doSnapshotCleanup(backupDone bool) (err error) {
+	defer func() {
+		if err != nil {
+			errMessage := errors.Wrapf(err, "failed to clean up old snapshots of volume %v", job.volumeName).Error()
+			job.eventCreate(corev1.EventTypeWarning, constant.EventReasonFailedDeleting, errMessage)
```

Comment:
Need to handle the returned error from `eventCreate` but it will be intended to ignore with warning log only.

Reply:
Add `Warn` log if the error is not `nil` from `job.eventCreate`.
Thanks